lib/ansible/modules/network/cloudengine/ce_vxlan_vap.py

```diff
@@ -0,0 +1,950 @@
+#!/usr/bin/python
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = """
+---
+module: ce_vxlan_vap
+version_added: "2.3"
+short_description: Manages VXLAN virtual access point.
+description:
+    - Manages VXLAN Virtual access point.
+author: QijunPan (@CloudEngine-Ansible)
+options:
+    bridge_domain_id:
+        description:
+            - Specifies a bridge domain ID.
+              The value is an integer ranging from 1 to 16777215.
+        required: false
+        default: null
+    bind_vlan_id:
+        description:
+            - Specifies the vlan binding to a BD(Bridge Domain).
+              The value is an integer ranging ranging from 1 to 4094.
+        required: false
+        default: null
+    l2_sub_interface:
+        description:
+            - Specifies an Sub-Interface full name, i.e. "10GE1/0/41.1".
+              The value is a string of 1 to 63 case-insensitive characters, spaces supported.
+        required: false
+        default: null
+    encapsulation:
+        description:
+            - Specifies an encapsulation type of packets allowed to pass through a Layer 2 sub-interface.
+        choices: ['dot1q', 'default', 'untag', 'qinq', 'none']
+        required: false
+        default: null
+    ce_vid:
+        description:
+            - When C(encapsulation) is 'dot1q', specifies a VLAN ID in the outer VLAN tag.
```

Comment:
`I(encapsulation)`

`I(...) for option name`
http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#formatting-options

This applies to multiple places in this PR, and multiple PRs